when running job from releng-testing it is failed in Argus submit and finish command

When running job from releng-testing it is failed in Argus submit and finish command:
https://jenkins.scylladb.com/job/scylla-master/job/releng-testing/job/azure-image/326/

with an error:
```
11:03:08  + docker run -w /jenkins/workspace/scylla-master/releng-testing/azure-image/scylla-pkg/scripts/argus-client -v /jenkins:/jenkins -v /home/jenkins:/home/jenkins scylladb/argus-client-generic:latest argus-client-generic submit --api-key **** --id ccc72b48-a424-4acb-aa65-bcfc1362606d --build-id scylla-master/releng-testing/azure-image --build-url https://jenkins.scylladb.com/job/scylla-master/job/releng-testing/job/azure-image/326/ --scylla-version 6.3.0~dev-0.20241118.572b005774b8 --started-by annamikhlin
11:03:09  Traceback (most recent call last):
11:03:09    File "/usr/local/bin/argus-client-generic", line 8, in <module>
11:03:09      sys.exit(cli())
11:03:09               ^^^^^
11:03:09    File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
11:03:09      return self.main(*args, **kwargs)
11:03:09             ^^^^^^^^^^^^^^^^^^^^^^^^^^
11:03:09    File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1078, in main
11:03:09      rv = self.invoke(ctx)
11:03:09           ^^^^^^^^^^^^^^^^
11:03:09    File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
11:03:09      return _process_result(sub_ctx.command.invoke(sub_ctx))
11:03:09                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
11:03:09    File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
11:03:09      return ctx.invoke(self.callback, **ctx.params)
11:03:09             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
11:03:09    File "/usr/local/lib/python3.11/site-packages/click/core.py", line 783, in invoke
11:03:09      return __callback(*args, **kwargs)
11:03:09             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
11:03:09    File "/usr/local/lib/python3.11/site-packages/argus/client/generic/cli.py", line 26, in submit_run
11:03:09      client.submit_generic_run(build_id=build_id, run_id=id, started_by=started_by, build_url=build_url, scylla_version=scylla_version)
11:03:09    File "/usr/local/lib/python3.11/site-packages/argus/client/generic/client.py", line 23, in submit_generic_run
11:03:09      self.check_response(response)
11:03:09    File "/usr/local/lib/python3.11/site-packages/argus/client/base.py", line 68, in check_response
11:03:09      raise ArgusClientError(
11:03:09  argus.client.base.ArgusClientError: ('API Error encountered using endpoint: POST /api/v1/client/testrun/generic/submit', 'DoesNotExist')
```

The same process passed when running from staging:
https://jenkins.scylladb.com/job/scylla-staging/job/Anna/job/azure-image/23/
```
11:16:23  + docker run -w /jenkins/workspace/scylla-staging/Anna/azure-image/scylla-pkg/scripts/argus-client -v /jenkins:/jenkins -v /home/jenkins:/home/jenkins scylladb/argus-client-generic:latest argus-client-generic submit --api-key **** --id f10160c8-192f-4abc-813e-91fa37c65b7a --build-id scylla-staging/Anna/azure-image --build-url https://jenkins.scylladb.com/job/scylla-staging/job/Anna/job/azure-image/23/ --scylla-version 6.3.0~dev-0.20241118.572b005774b8 --started-by annamikhlin
```
both runnig from the same branch with the same params.

interestingly, we don't have logs from argus during that time:
```
Nov 27 09:03:24 ip-10-0-2-112 bash[1114542]: [pid: 1114542|app: 0|req: 50324/483474] 176.120.96.40 () {58 vars in 2844 bytes} [Wed Nov 27 09:03:24 2024] GET /api/v1/client/sct/cf1688b2-5cce-4514-a096-b86c1121b8c6/junit/get_all => generated 30 bytes in 6 msecs (HTTP/1.1 200) 3 h>
...skipping...
Nov 27 09:03:25 ip-10-0-2-112 bash[1114542]: [ERROR] <5.29.124.170 - https://argus.scylladb.com/api/v1/client/testrun/generic/ccc72b48-a424-4acb-aa65-bcfc1362606d/finalize - api.client_api.run_finalize> - error_handlers::handle_api_exception - Exception in api.client_api.run_fi>
Nov 27 09:03:25 ip-10-0-2-112 bash[1114542]: Traceback (most recent call last):
```

--- a/argus/backend/plugins/driver_matrix_tests/model.py
+++ b/argus/backend/plugins/driver_matrix_tests/model.py
@@ -8,6 +8,7 @@ from typing import Literal, TypedDict
 from uuid import UUID
 from xml.etree import ElementTree
 from cassandra.cqlengine import columns
+from cassandra.cqlengine.models import Model
 from argus.backend.db import ScyllaCluster
 from argus.backend.models.web import ArgusRelease
 from argus.backend.plugins.core import PluginModelBase
@@ -412,7 +413,10 @@ class DriverTestRun(PluginModelBase):
 
     def submit_product_version(self, version: str):
         self.scylla_version = version
-        new_assignee = self.get_assignment(version)
+        try:
+            new_assignee = self.get_assignment(version)
+        except Model.DoesNotExist:
+            new_assignee = None
         if new_assignee:
             self.assignee = new_assignee
 

--- a/argus/backend/plugins/generic/model.py
+++ b/argus/backend/plugins/generic/model.py
@@ -41,7 +41,10 @@ class GenericRun(PluginModelBase):
         pattern = re.compile(r"((?P<short>[\w.~]+)-(?P<build>(0\.)?(?P<date>[0-9]{8,8})\.(?P<commit>\w+).*))")
         if match := pattern.search(version):
             self.scylla_version = match.group("short")
-            new_assignee = self.get_assignment(match.group("short"))
+            try:
+                new_assignee = self.get_assignment(match.group("short"))
+            except Model.DoesNotExist:
+                new_assignee = None
             if new_assignee:
                 self.assignee = new_assignee
             self.set_full_version(version)

--- a/argus/backend/plugins/sct/testrun.py
+++ b/argus/backend/plugins/sct/testrun.py
@@ -216,7 +216,10 @@ class SCTTestRun(PluginModelBase):
 
     def submit_product_version(self, version: str):
         self.scylla_version = version
-        new_assignee = self.get_assignment(version)
+        try:
+            new_assignee = self.get_assignment(version)
+        except Model.DoesNotExist:
+            new_assignee = None
         if new_assignee:
             self.assignee = new_assignee
 

--- a/argus/backend/plugins/sirenada/model.py
+++ b/argus/backend/plugins/sirenada/model.py
@@ -61,7 +61,10 @@ class SirenadaRun(PluginModelBase):
 
     def submit_product_version(self, version: str):
         self.scylla_version = version
-        new_assignee = self.get_assignment(version)
+        try:
+            new_assignee = self.get_assignment(version)
+        except Model.DoesNotExist:
+            new_assignee = None
         if new_assignee:
             self.assignee = new_assignee
 

